src/app/machines/views/MachineList/MachineListTable/MachineListTable.tsx

```diff
@@ -829,6 +832,36 @@ export const MachineListTable = ({
     [hiddenColumns, showActions]
   );
 
+  const selectionState = useMemo(() => {
+    if (selectedCount > 0) {
+      return [
+        {
+          className: "select-notification",
+          key: "select-info",
+          columns: [
+            {
+              colSpan: columns.length - hiddenColumns.length,
+              content: (
+                <MachineListSelectedCount
+                  filter={filter}
+                  machineCount={machineCount}
+                  selectedCount={selectedCount}
+                />
+              ),
+            },
+          ],
+        },
+      ];
+    } else {
+      return [];
+    }
+  }, [filter, hiddenColumns.length, machineCount, selectedCount]);
+
+  const renderRows = useMemo(
```

Comment:
```suggestion
  const machineRows = useMemo(
```
`render` implies it's a render function